### DISCUSSION TITLE
license question/viewtracker payware

### DISCUSSION THREAD

**Discussion by Bassemann87** [opentrack/opentrack#1578] [General]
Hi! 
Im a Opentrack user and today i found a payed software called Viewtracker on Steam. It looks like the developer Ripped Opentrack and change the name of the software.

 There is also a "demo" of the software to download on the steampage.

Is it allowed to do this? It just feels wrong that people dont use this instead.

Link to the software on steam:
https://store.steampowered.com/app/929270/ViewTracker/

Thanks for answers! // Sebastian S.

**Comment by Pieloth** [comment]
AFAIK, Viewtracker project was the first to introduce a Webcam face detection for Opentrack in 2018, the dev proposed his packaged solution and sold it on Steam
However, the project died a while back, no more updates or communications, and it does not work very well so far
Looks like another abandonware 
Compared with today's Neuralnet + Opentrack, Viewtracker appears to be an old outdated face tracking solution, with weak results especially if you wear googles or under low light, per own experience
Neuralnet + Opentrack offer much better results
I don't know if it sells much today, the discussion forum warns about the product state and usage, and points towards the Opentrack solution instead

**Reply by Bassemann87** [reply]
Ok thanks for clarifying this :)
